.github/workflows/cfn-analysis.yml

```diff
@@ -26,21 +26,11 @@ jobs:
           node-version: 14
       - name: Install npm dependencies
         run: yarn install
-      - name: Package main sls into cfn for analysis
+      - name: synthesize cdk template for cfn for analysis
         run: |
-          yarn run serverless-package --stage dev --region us-west-2 --useHapiValidator true --conceal
-      - name: cfn_nag on main serverless
+          yarn cdk synth -c issuerEndpoint='test' -c oAuth2ApiEndpoint='test' -c patientPickerEndpoint='test' -c require-approval=never
```

Comment:
Quick thing: this should be `--require-approval=never`, not `-c require-approval=never`. Other than that PR looks good to me!